tests/kas_with_instance.spec.ts

```diff
@@ -183,3 +183,75 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_4kafka.py test_edit_topic_properties_after_creation
+test('edit topic properties after creation', async ({ page }) => {
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName);
+
+  const instanceLinkSelector = page.getByText(testTopicName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Edit topic configuration').click();
+
+  // store previous num partitions value to compare afterwards
+  const numPartitionsBefore: string = await page.locator('input[name="num-partitions"]').getAttribute('value');
+
+  for (let i = 0; i < 2; i++) {
+    await page.locator('button[name="num-partitions"]').nth(1).click();
+  }
+  await page.locator('button[name="num-partitions"]').first().click();
+
+  for (let i = 0; i < 2; i++) {
+    await page.locator('button[name="retention-ms"]').nth(1).click();
+  }
+  await page.locator('button[name="retention-ms"]').first().click();
+
+  await page.locator('button:has-text("days")').click();
+  await page.getByText('hours').click();
```

Comment:
This looks to me like a very vague selector.

Reply:
there is a problem with the units select as it is not a select, but a button (the outcome from Patternfly)

Reply:
so `button:hast-text("hours")`?

Reply:
yes, there is no `select` to locate for: 
![image](https://user-images.githubusercontent.com/100594859/213180870-eae1d46c-69ce-4f0f-a4d0-1a89cd1f27a2.png)

Reply:
I think there is a way (not sure if this works for my suggestion) to pick a specific element with text somewhere inside so button with hours (`button >> has-text(hours)` maybe?) should work

Reply:
I have tried some workarounds but in the end I just stuck with the PW debugger outcome at recording which is assured to work

Reply:
As long as it will work and some else will approve (with more experience than I have), I can live with it. However, I think the selector could be better. Feel free to resolve this.

Reply:
Replaced for `await page.locator('button', { hasText: 'hours' }).click();`